blocks/graph.py

```diff
@@ -161,8 +161,37 @@ def replace(self, replacements):
             substitutes.
 
         """
-        return ComputationGraph(theano.clone(self.outputs,
-                                             replace=replacements))
+        # Due to bug in theano we have to make one replacement in time
```

Comment:
As far I understood when discussing this with @harmdevries89, this is actually expected behavior; it's the doc string that is wrong I guess? Anyway, there is no Theano issue that I am aware of. Maybe @nouiz would be interested in moving this functionality of being able to replace multiple, dependent variables from Blocks to Theano eventually though?